guides/common/modules/proc_registering-project-as-a-client-of-keycloak.adoc

```diff
@@ -22,16 +22,16 @@ You can also register two different {Project} clients to {Keycloak} if you want
 # {project-package-install} mod_auth_openidc keycloak-httpd-client-install python3-lxml
 ----
 
-. Register {Project} to {Keycloak} as a client.
+. Register {Project} to {keycloak} as a client.
 Note that you the registration process for logging in using the web UI and the CLI are different.
-You can register two clients {Project} clients to {Keycloak} to be able to log in to {Project} from the web UI and the CLI.
+You can register two clients {Project} clients to {keycloak} to be able to log in to {Project} from the web UI and the CLI.
 +
 * If you want you users to authenticate to {Project} using the web UI, create a client as follows:
 +
 [options="nowrap", subs="verbatim,quotes,attributes"]
 ----
 # keycloak-httpd-client-install --app-name foreman-openidc \
---keycloak-server-url "_https://{Keycloak-short}.example.com_" \
+--keycloak-server-url "_https://{keycloak-example-com}_" \
 --keycloak-admin-username "_admin_" \
 --keycloak-realm "_{Project}_Realm_" \
```

Comment:
```suggestion
--keycloak-realm "_{keycloak-realm}_" \
```
I wonder if this is supposed to be the same value. Also other occurences.

Reply:
Also, it would be nice to clarify allowed characters for this field. If it's going to be a part of the URL, there must be limitations.

Reply:
Addressing this would be a much bigger scope creep than I'm comfortable with in this PR. In this PR, I'm only making changes necessary to replace the currently-used attributes `keycloak-short` and `Keycloak` -- because these attrs have actually stopped working for our d/s now that we're dealing with Red Hat build of Keycloak in addition to Red Hat Single Sign On.

So, no change for this comment thread @Lennonka because if there is indeed a need to create a new attribute for a Project/Keycloak realm, I'll do that in a subsequent PR where I edit and test the procedures.